assemblyline/odm/models/submission.py

```diff
@@ -1,32 +1,37 @@
 from __future__ import annotations
+
 import hashlib
+
 from assemblyline import odm
 from assemblyline.common import forge
+
 Classification = forge.get_classification()
 
 SUBMISSION_STATES = ['failed', 'submitted', 'completed']
 DEFAULT_SRV_SEL = ["Filtering", "Antivirus", "Static Analysis", "Extraction", "Networking"]
 DEFAULT_RESUBMIT = []
 
 
-@odm.model(index=True, store=False, description="File Model of Submission")
+@odm.model(index=True, store=False, description="""Contains details about the individual files included in the submission, including their names, sizes, and SHA256 hashes.
+""")
 class File(odm.Model):
-    name = odm.Keyword(copyto="__text__", description="Name of the file")
-    size = odm.Optional(odm.Integer(), description="Size of the file in bytes")
-    sha256 = odm.SHA256(copyto="__text__", description="SHA256 hash of the file")
+    name = odm.Keyword(copyto="__text__", description="The original name of the file as it was submitted.")
+    size = odm.Optional(odm.Integer(), description="The file's size in bytes.")
+    sha256 = odm.SHA256(copyto="__text__", description="The SHA256 cryptographic hash of the file, serving as a unique identifier for the file's content.")
 
 
-@odm.model(index=False, store=False, description="Service Selection Scheme")
+@odm.model(index=False, store=False, description="""Outlines the services selected for analysis, any excluded services, and any additional services that should be used in the case of rescan or resubmit actions.
+""")
 class ServiceSelection(odm.Model):
-    selected = odm.List(odm.Keyword(), default=DEFAULT_SRV_SEL, description="List of selected services")
-    excluded = odm.List(odm.Keyword(), default=[], description="List of excluded services")
+    selected = odm.List(odm.Keyword(), default=DEFAULT_SRV_SEL, description="Services selected to process the submission, which determine the scope of analysis.")
+    excluded = odm.List(odm.Keyword(), default=[], description="Services explicitly excluded from processing the submission, bypassing their analysis.")
     rescan = odm.List(
         odm.Keyword(),
         default=[],
-        description="List of services to rescan when initial run scores as malicious")
+        description="Services to be used for a rescan if the submission's initial results are deemed malicious.")
     resubmit = odm.List(odm.Keyword(), default=DEFAULT_RESUBMIT,
-                        description="Add to service selection when resubmitting")
-    runtime_excluded = odm.List(odm.Keyword(), default=[], description="List of runtime excluded services")
+                        description="Additional services that are added to the selection when a submission is resubmitted.")
+    runtime_excluded = odm.List(odm.Keyword(), default=[], description="**TODO**: **Original**:List of runtime excluded services  **Generated**:Services that are excluded during the submission's runtime, possibly due to dynamic decision-making.")
```

Comment:
This field definitely doesn't get used and should be removed

Reply:
Ok, I will delete that field (`runtime_excluded`) from the model.